Rakefile

```diff
@@ -1,3 +1,5 @@
+$in_rake_task = true
```

Comment:
I think I got biased while looking around stack overflow and now I can't come up with a nicer solution than this, any ideas @flbulgarelli ?

Reply:
You may use `defined? Rake.application`